src/main/java/org/opensearch/knn/indices/ModelMetadata.java

```diff
@@ -123,6 +126,9 @@ public ModelMetadata(
         this.state = new AtomicReference<>(Objects.requireNonNull(modelState, "modelState must not be null"));
         this.timestamp = Objects.requireNonNull(timestamp, "timestamp must not be null");
         this.description = Objects.requireNonNull(description, "description must not be null");
+        if (this.description.contains(",")) {
+            throw new IllegalArgumentException("Model description cannot contain any commas: ','");
```

Comment:
nit: may be "model" instead of "Model"? i see every other exception message doesn't start with caps.

Reply:
All of the null check exceptions are lower case, but the other IllegalArgumentException earlier in the method that validates the max dimensions starts with a cap.